HesoyamHospital/WebApplicationTests/Integration/Scheduling/AppointmentSchedulingTests.cs

```diff
@@ -66,6 +67,38 @@ public AppointmentSchedulingTests(WebApplicationFactory<Startup> factory)
         //    response.StatusCode.ShouldBeEquivalentTo(expectedStatusCode);
         //}
 
+        //[Theory]
```

Comment:
Is this a temporary commented section or it has no further purpose in the project?
If it has no further purpose, delete it and the PR will be approved.